pkg/controller/account/account_controller.go

```diff
@@ -116,8 +119,15 @@ type ReconcileAccount struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("Reconciling Account")
+	start := time.Now()
+	reqLogger := log.WithValues("Controller", controllerName, "Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.Info("Reconciling")
```

Comment:
Just making sure I understand:

By setting up `WithValues` with `"Controller", controllerName`, we're getting the name of the controller included in every log line, including this one. (I think that's what you're acking when you say, "I know the controller name is logged in the line before this.") So this log line looks like:

`{"level":"info","ts":1594250062.9127512,"logger":"controller_account","msg":"Reconciling","Controller":"account","Request.Namespace":"aws-account-operator","Request.Name":"osd-creds-mgmt-4gqrvp"}`

So are you saying we should leave the redundant "Account" in the message because

`{"level":"info","ts":1594250062.9127512,"logger":"controller_account","msg":"Reconciling Account","Controller":"account","Request.Namespace":"aws-account-operator","Request.Name":"osd-creds-mgmt-4gqrvp"}`

would be more readable?

Reply:
I'm :+1: for adding the data into the "metadata" of the logger instead of the log messages itself, like what @2uasimojo has proposed.  Not sure if there's an unknown drawback to that, though?

Reply:
@jharrington22 final ruling?

Reply:
What is nice about this specific message is when you are looking at the logs you know this marks the beginning of the reconcile loop for this particular run, I think it is useful for debugging purposes even though the metadata reports it already.

Reply:
I've found it useful in the past to identify that a controller is either hot looping or stalled.

Reply:
Fine to leave as is I just realized thanks @2uasimojo that the controller name is in the meta for whatever reason I thought we were removing the message entirely. :facepalm:

Reply:
Great, thanks @jharrington22.